README.md

```diff
@@ -1,60 +1,101 @@
-|50.teams-messaging-extensions-search     |  A Messaging Extension that accepts search requests and returns results.|[View][cs#50]|[View][js#50]          |              |     |     |[View][py#50]
-|51.teams-messaging-extensions-action     |  A Messaging Extension that accepts parameters and returns a card.  Also, how to receive a forwarded message as a parameter in a Messaging Extension.|[View][cs#51]|[View][js#51]          |              |     |     |[View][py#51]
-|52.teams-messaging-extensions-search-auth-config     | A Messaging Extension that has a configuration page, accepts search requests and returns results after the user has signed in.|[View][cs#52]|[View][js#52]          |              |     |     |
-|53.teams-messaging-extensions-action-preview     | Demonstrates how to create a Preview and Edit flow for a Messaging Extension.|[View][cs#53]|[View][js#53]          |              |     |     |[View][py#53]
-|54.teams-task-module     | Demonstrates how to retrieve a Task Module, and values from cards in the Task Module, for a Messaging Extension.|[View][cs#54]|[View][js#54]          |              |     |     |[View][py#54] 
-|55.teams-link-unfurling     | A Messaging Extension that performs link unfurling.|[View][cs#55]|[View][js#55]          |              |     |     |[View][py#55]
-|56.teams-file-upload     | Demonstrates how to obtain file consent, and upload files to Teams from a bot. Also, how to receive a file sent to a bot.|[View][cs#56]|[View][js#56]          |              |     |     |[View][py#56]
-|57.teams-conversation-bot     | Demonstrates various features of bots on Teams: message all members in a Team or Channel, @mention a user from a bot, update previously sent messages, etc. |[View][cs#57]|[View][js#57]          |              |     |     |[View][py#57]
-|70.styling-webchat     | This sample shows how to create a web page with custom Web Chat component.|         |          |              |  [View][es#70] |     |
-|80.skills-simple-bot-to-bot     | This sample shows how to connect a skill to a skill consumer.| [View][cs#80] | [View][js#80]       |              |   |     |[View][py#80]
+## Sample lists
+
+Samples are designed to illustrate scenarios you'll need to implement to build great bots!
+
+- [Bot Essentials](#bot-essentials)
+- [Bot Advanced](#bot-advanced)
+- [Auth Samples](#auth-samples)
+- [QnAMaker Samples](#qnamaker-samples)
+- [Teams Samples](#teams-samples)
+- [Skills Samples](#skills-samples)
+
+### Bot Essentials
+
+| Sample Name           | Description                                                                      | .NET    | JavaScript   | Python  |
+|:----------------------|:---------------------------------------------------------------------------------|:--------|:-------------|:--------|
+|2.echo-bot             | Demonstrates how to receive and send messages.                                 |[Core][cs#2] |[JS][js#2], [TS][ts#2]|[Python][py#2]

```

Comment:
I thought the suggestion had been to create one column for sample number and one for a friendly short name:
```suggestion
| Number | Name | Description | .NET | JavaScript | Python  |
| :----- | :--- | :----------- | :--- | :-------- | :----- |
| 2 | Echo bot | Demonstrates how to receive and send messages. | [.NET Core][cs#2] | [JavaScript][js#2], [TypeScript][ts#2] | [Python][py#2]

```
And so on

Reply:
And that makes sense... Except for the convenient duplication of 

1.console-echo and 1.browser-echo

Reply:
Ack. :(